src/odemis/acq/test/path_test.py

```diff
@@ -26,8 +26,8 @@
 import odemis
 from odemis.acq import path, stream
 from odemis.acq.path import ACQ_QUALITY_BEST, ACQ_QUALITY_FAST
-from odemis.util import test
-from odemis.util.test import assert_pos_almost_equal
+from odemis.util import testing
+from odemis.util.testing import assert_pos_almost_equal
```

Comment:
can you update the code that uses `assert_pos_almost_equal` to use it as `testing.assert_pos_almost_equal` and then remove this line?